src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/ClusterServiceMonitor.scala

```diff
@@ -0,0 +1,105 @@
+package org.broadinstitute.dsde.workbench.leonardo.monitor
+
+import akka.actor.{Actor, Props, Timers}
+import org.broadinstitute.dsde.workbench.leonardo.Metrics
+import com.typesafe.scalalogging.LazyLogging
+import org.broadinstitute.dsde.workbench.google.GoogleProjectDAO
+import org.broadinstitute.dsde.workbench.leonardo.config.ClusterServiceConfig
+import org.broadinstitute.dsde.workbench.leonardo.dao.{JupyterDAO, WelderDAO}
+import org.broadinstitute.dsde.workbench.leonardo.dao.google.GoogleDataprocDAO
+import org.broadinstitute.dsde.workbench.leonardo.db.DbReference
+import org.broadinstitute.dsde.workbench.leonardo.model.Cluster
+import org.broadinstitute.dsde.workbench.leonardo.monitor.ClusterServiceMonitor.{DetectClusterStatus, JupyterStatus, Status, TimerKey, WelderStatus}
+import org.broadinstitute.dsde.workbench.model.google.GoogleProject
+import org.broadinstitute.dsde.workbench.newrelic.NewRelicMetrics
+
+import scala.concurrent.Future
+
+object ClusterServiceMonitor {
+
+
+  val newRelic: NewRelicMetrics = Metrics.newRelic
```

Comment:
Is this `val` used? It looks like `ClusterServiceMonitor` uses the `NewRelicMetrics` passed to its constructor.